devicetypes/smartthings/zwave-switch-generic.src/zwave-switch-generic.groovy

```diff
@@ -57,6 +57,9 @@ metadata {
 		fingerprint mfr: "0312", prod: "FF00", model: "FF07", deviceJoinName: "Minoston Outdoor Smart Plug", ocfDeviceType: "oic.d.smartplug"
 		fingerprint mfr: "0312", prod: "FF00", model: "FF06", deviceJoinName: "Minoston Smart Plug 1CH", ocfDeviceType: "oic.d.smartplug"
 		fingerprint mfr: "0312", prod: "FF00", model: "FF01", deviceJoinName: "Minoston on/off Toggle Switch", ocfDeviceType: "oic.d.smartplug"
+		fingerprint mfr: "0312", prod: "C000", model: "C003", deviceJoinName: "Evalogik Outdoor Smart Plug", ocfDeviceType: "oic.d.smartplug"
+		fingerprint mfr: "0312", prod: "FF00", model: "FF03", deviceJoinName: "Minoston Smart On/Off Switch"
+		fingerprint mfr: "0312", prod: "C000", model: "CO05", deviceJoinName: "Evalogik Mini Outdoor Smart Plug"
```

Comment:
Based on the product name it seems this should specify `ocfDeviceType: "oic.d.smartplug"` so it has the plug icon instead of the switch icon. What do you think?

Reply:
agreed

Reply:
Looks like they all should, since they seem to just be different regions of devices that we already have that are using the smartplug icon.

Reply:
Yes ,I have corrected it.Thank you.